### ISSUE TITLE
Understanding Loss

### ISSUE THREAD

**Issue by sheshap** [MingyeXu/GS-Net#2]
I downloaded your code and trained using the instructions you had given.

I got 93.1%+ accuracy as shown below which is pretty good. 

Test 413, loss: 1.436297, test acc: 0.930713, test avg acc: 0.895570

However, I want to understand why is the loss so high(1.436) compared to pointnet++(had tried both TF and Pytorch implementations) whose accuracy was lesser and loss was always less than 0.5.
